### DISCUSSION TITLE
How to access "Visual File History" feature?

### DISCUSSION THREAD

**Discussion by mushlih-almubarak** [gitkraken/vscode-gitlens#2463] [Q&A] [answered]
Hi, I have activated my "GitLens+ Pro" subscription, how do I access the "Visual File History" feature as mentioned [here](https://github.com/gitkraken/vscode-gitlens/discussions/2170#:~:text=New%20Commit%20Graph-,Visual%20File%20History,-Worktrees)?

**Comment by d13** [comment, marked as answer]
Hi @mushlih-almubarak! Here's what to do.

Make sure your bottom panel is open:

![Screen Shot 2023-01-23 at 9 40 59 AM](https://user-images.githubusercontent.com/293282/214070072-bed53a8a-b246-447d-a583-6e1ec0853755.png)

Click the "GitLens" tab:

![Screen Shot 2023-01-23 at 9 41 27 AM](https://user-images.githubusercontent.com/293282/214070121-1a420791-4300-48ab-babb-618b8b93cbb1.png)

As you're view files open in the editor, the visualization will appear:

![Screen Shot 2023-01-23 at 9 41 51 AM](https://user-images.githubusercontent.com/293282/214070394-57dd3ed5-98d6-44d6-a53e-07ca7939ca7c.png)

If a file hasn't been changed in a while, you may have to adjust the "Timeframe":

![Screen Shot 2023-01-23 at 9 55 12 AM](https://user-images.githubusercontent.com/293282/214071113-7906fbdb-4c91-4eac-b2d8-b3292b5fe7a6.png)

**Reply by mushlih-almubarak** [reply]
Thanks! It worked!

**Reply by eamodio** [reply]
You can also run the `Show Visual File History View` command from the Command Palette.

**Reply by mushlih-almubarak** [reply]
> You can also run the `Show Visual File History View` command from the Command Palette.

Thanks!
